### ISSUE TITLE
BLE don't work in ZHA mode (component v0.8)

### ISSUE THREAD

**Issue by Sleinous** [AlexxIT/XiaomiGateway3#94]
These Xiaomi temperature and humidity sensors have been working fine since early releases of this integration. However end of last week all 4 thermometers dropped off the radar in Home Assistant and i'm unable to add them back in now (even with the new method of re-adding Gateway & token etc)

All 4 thermometers (BLE LYWSDCGQ/01ZM) are correctly detected in Mi Home app and also seen by Xiaomi Gateway 3 BLE Mesh

I didn't notice the exact date these stopped working so not sure if it's releated to a recent update of inetgration, anyone else experiencing this issue now ?

(Xiaomi Gateway 3 FW is still 1.4.6_0012)

**Comment by AlexxIT** [comment]
You can enable debug mode and show logs.

**Comment by Sleinous** [comment]
This is all i'm seeing in Xiaomi Gateway 3 logs each time : 

2020-11-02 16:08:37  DEBUG    gateway3      192.168.2.81 | Prepare Gateway
2020-11-02 16:08:37  DEBUG    gateway3      192.168.2.81 | Run socat
2020-11-02 16:08:45  DEBUG    main          Loaded from MiCloud 10 devices

**Comment by AlexxIT** [comment]
Have you later something?

**Comment by Sleinous** [comment]
Unfortunately not, still same three lines

**Comment by Sleinous** [comment]
Resolved : Fully uninstalled ZHA & Xiaomi Gateway 3 from HACS, reboot, reinstalled 0.7.0, all good again. Have attempted upgrade to 0.8.1 now without issues
Not sure what happened

**Comment by SpeedBlack** [comment]
Hello all,

Thanks Sleinous !
I have exactly the same problem! But as I started with this integration I didn't understand what was wrong! I uninstalled version 0.8.0 and installed 0.7.0 and it works.

Home Assistant : 0.117.1
Gateway : v1.4.6_0012

**Comment by Sleinous** [comment]
No worries, i'm currently back tracking to try & help to see when this issue appeared - I just crossed off 0.8.0 & 0.8.1, am now testing 0.7.2

**Comment by AlexxIT** [comment]
Sorry. My bad. With component v0.8.0-0.8.1 in ZHA mode - BLE don't work! Will fix it in next release.

**Comment by Sleinous** [comment]
No worries :)

**Comment by SpeedBlack** [comment]
No worries too :).
Thanks for your job !!

**Comment by froggyfly** [comment]
Emmm.... I'd like to say, My BLE at ZHA mode works good now, 
Like Sleinous and SpeedBlack said, BLE at first didn't work after upgraded to version 0.8 on ZHA mode,
Try to restart and reinstall, no used.
I'm not sure which operations let me solve this problem,
I tried to recall the last operation, maybe this can be useful,

I first deleted gateway3 and Zha integration,
Restart the homeassistant and gateway,
Added gateway3 integration at normal mode, BLE device work well. Then I delete this integration,
I add it to Zha mode again, 
Add the Zha integration, the third-party ZigBee devices that have been added before will appear, and the BLE device work.


I'm not sure it's completely useful, but my gateway is running in V 0.8, Zha mode, and ble devices are running normally
